spec/linked_data_fragments/service_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+require 'linked_data_fragments/service'
+require 'linked_data_fragments/repository'
+
+describe LinkedDataFragments::Service do
+  subject { described_class.instance }
+  
+  before do
+    # @todo: REMOVE THIS HACKY STUB
```

Comment:
I intended later, but would welcome push back.

I think the hacky stub is slightly better than the hard coded dependency. It's also in-line with what's done in the rest of the code base. If we decide fix it prior to merge, it might still be a separate PR replacing many similar stubs throughout.